AvalonStudio/AvalonStudio/Controls/ExtensionManagerDialog/ExtensionManagerDialogViewModel.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Globalization;
+using Avalonia;
+using Avalonia.Markup;
+using Avalonia.Media.Imaging;
+using Avalonia.Markup.Xaml.Converters;
+using Avalonia.Platform;
+using AvalonStudio.Extensibility;
+using AvalonStudio.Extensibility.Dialogs;
+using ReactiveUI;
+using System.Collections.ObjectModel;
+
+namespace AvalonStudio.Controls.ExtensionManagerDialog
+{
+    public class ExtensionManagerDialogViewModel : ModalDialogViewModelBase
+    {
+        private ExtensionManager _extensionManager;
+
+        private ObservableCollection<IExtensionManifest> _installedExtensions;
+        private IExtensionManifest _selectedExtension;
+
+        public ExtensionManagerDialogViewModel()
+            : base("Extensions")
+        {
+            _extensionManager = IoC.Get<ExtensionManager>();
+            _installedExtensions = new ObservableCollection<IExtensionManifest>(_extensionManager.GetInstalledExtensions());
+        }
+
+        public ObservableCollection<IExtensionManifest> InstalledExtensions => _installedExtensions;
+
+        public IExtensionManifest SelectedExtension
+        {
+            get => _selectedExtension;
+            set => this.RaiseAndSetIfChanged(ref _selectedExtension, value);
+        }
+    }
+
+    public class BitmapValueConverter : IValueConverter
```

Comment:
I think it could go inside a folder named `Converters` but probably don't need a common place if Avalonia is going to support this soon.

Otherwise maybe Avalonstudio.Extensibility.Converters? maybe